Classes/Domain/Import/Entity/Place.php

```diff
@@ -168,7 +170,10 @@ public function setGeo(Geo $geo): void
      */
     public function getOpeningHoursSpecification(): array
     {
-        return $this->openingHours;
+        $today = GeneralUtility::makeInstance(Context::class)->getPropertyFromAspect('date', 'full');
```

Comment:
I don't like the code as it is duplicated from Model/Frontend and uses makeInstance().